### PR TITLE
Focus on code split

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
- One-line PR description:

- Issue link:

- Other comments:

### CODE REVIEW

File: keps/sig-scheduling/3902-decoupled-taint-manager/README.md [hunk 1]
```diff
@@ -60,35 +57,38 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
...
 * Actual extension and improvement of the current `TaintManager`.
 
 ## Proposal
 
 ### User Stories (Optional)
 
-#### Story 1
+#### Story
+
+While this split is more of a kubernetes developer focused change, the side-effect of this change is that it allows cluster-admins to develop custom `TaintManager`s and disable the default `TaintManager`.
```
Comment (Huang-Wei) [inline keps/sig-scheduling/3902-decoupled-taint-manager/README.md]: side-effect sounds more like a negative effect, maybe reword to "by-product" or "unintended positive effect".

File: keps/sig-scheduling/3902-decoupled-taint-manager/README.md [hunk 2]
```diff
@@ -60,35 +57,38 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
...
 
 ### User Stories (Optional)
 
-#### Story 1
+#### Story
+
+While this split is more of a kubernetes developer focused change, the side-effect of this change is that it allows cluster-admins to develop custom `TaintManager`s and disable the default `TaintManager`. 
 
-An operator of stateful workloads that use local persistent volume can disable the default taint-manager and use a custom controller instead. When a node is tainted with `NoExecute` , the custom controller won’t delete the running stateful pods on the tainted node until the data is safe,  e.g., after the application manager successfully checkpoints the data in or migrates the data to a safe place. 
+The default `TaintManager` because of the implementation rigidity may not be suitable for every cluster operator. We except users to come up with custom `TaintManager`s and this KEP may exacerbate the problem of `bring your own taint manager`. However, discussing the use-cases for extending `TaintManager` or custom `TaintManger`s is beyond the scope of this KEP.
```
Comment (Huang-Wei) [inline keps/sig-scheduling/3902-decoupled-taint-manager/README.md]: The default `TaintManager`, because of the implementation rigidity, may not be suitable for every cluster operator.

**Review by Huang-Wei — COMMENTED** [review]
I like the idea of simplifying stories so as to concentrate on the structure refactoring.

Some nits.

**Review by yuanchen8911 — APPROVED** [review]
/lgtm

**Comment by yuanchen8911** [comment]
@ravisantoshgudimetla , can you take a look at Wei's comments. Otherwise, its lgtm.
